### PR TITLE
Minor correction of docstring listing truth dataframe columns

### PR DESCRIPTION
**Pull request by gavinevans**
Addresses https://github.com/metoppv/mo-blue-team/issues/207

Description
This PR makes a minor correction to a docstring listing the input columns for the truth dataframe.

Testing:
 - [x] Ran tests and they passed OK

### CODE REVIEW

**Review by lucyleeow — DISMISSED** [review]
LGTM! (Would it be worth updating [`calibration_data_ingestion`](https://github.com/metoppv/improver/blob/master/doc/source/extended_documentation/calibration/calibration_data_ingestion.rst) as well? )

File: improver/calibration/dataframe_utilities.py [hunk 1]
```diff
@@ -281,7 +281,7 @@ def _prepare_dataframes(
             DataFrame expected to contain the following columns: forecast,
             blend_time, forecast_period, forecast_reference_time, time,
             wmo_id, percentile, diagnostic, latitude, longitude, period,
-            height, cf_name, units. Any other columns are ignored.
+            height, cf_name and units. Any other columns are ignored.
```
Comment (lucyleeow) [inline improver/calibration/dataframe_utilities.py:284]: The docs say 'experiment' is also required? https://improver.readthedocs.io/en/latest/improver.calibration.dataframe_utilities.html?highlight=dataframe#forecast-dataframe

Comment (gavinevans) [reply improver/calibration/dataframe_utilities.py]: You're correct. I've added `experiment` to this list.

File: improver/calibration/dataframe_utilities.py [hunk 2]
```diff
@@ -281,7 +281,7 @@ def _prepare_dataframes(
             DataFrame expected to contain the following columns: forecast,
             blend_time, forecast_period, forecast_reference_time, time,
             wmo_id, percentile, diagnostic, latitude, longitude, period,
-            height, cf_name, units. Any other columns are ignored.
+            height, cf_name, units and experiment. Any other columns are ignored.
```
Comment (lucyleeow) [inline improver/calibration/dataframe_utilities.py:284]: It seems like `altitude` is required as well? From: https://github.com/metoppv/improver/blob/a5cb842c30e580f48c5aa6ad35fa8a58584b92fc/improver/calibration/dataframe_utilities.py#L360

Comment (gavinevans) [reply improver/calibration/dataframe_utilities.py]: Yes, you're correct. I've added `altitude` to the list.

**Review by bayliffe — APPROVED** [review]

**Comment by lucyleeow** [comment]
Hi @gavinevans, sorry yes all good for me. Thanks

**Pull request merged**